build.gradle

```diff
@@ -182,6 +182,10 @@ task runTestsWithTag(type: Test) {
         environment "TERRA_DOCKER_IMAGE", terraDockerImage
 
         if (testTag == "unit") {
+            // set which file in src/test/resources/testconfigs to use
+            String terraTestConfigFileName = terraServer.contains("broad") ? "broad.json" : "verily.json"
+            environment "TERRA_TEST_CONFIG_FILE_NAME", terraTestConfigFileName
```

Comment:
nit: Can you use a `systemProperty` for this instead of an env var? I think you'll only need to access this inside the test Java code.